### DISCUSSION TITLE
Open Kurviger route from Firefox

### DISCUSSION THREAD

**Discussion by mkrezdo** [devemux86/cruiser#493] [Q&A] [answered]
In earlier time (Kurviger) it was possible to transfere a route from the kurviger website directly to the navigation app if Firefox was used as browser.
The only thing to do was to have a planned route open, then push the 3 points, then "open in app" and then choose the app.
After that the exact route was imported as a new route.

Is it possible to enable this with Cruiser too?
![2024-04-11 23_01_22-Window](https://github.com/devemux86/cruiser/assets/137488443/76378e25-a0e5-4ae3-8814-6cc367dca0fd)
![Screenshot_20240411_225749_com huawei android internal app](https://github.com/devemux86/cruiser/assets/137488443/f7680756-2cbc-4eb6-870c-d4a03b5ccf89)

Greetings,
Marco

**Comment by devemux86** [comment, marked as answer]
Hi @mkrezdo,

Transferring routes this way through the browser only transfers the route **link**,
so a new route calculation is required in the app. It is better to:

- **Export** the route as kurviger **file** with the browser
- **Import** the kurviger file with Cruiser

Use Cruiser import menu or click the kurviger file with a file manager and Cruiser will be in the available apps.

**Comment by DIrgi** [comment]
@mkrezdo
Proposal:
If you use a cloud service i.e. Google Drive (or any other) - This is free up to 15GB which ich quite a lot of space for hundreds of gpx files. You can safe and share all your files between all of your hardware devices. You can even establish a much more better database structure then Kurviger offers with her implemented cloud. Nothing against Kurviger I love it but if you use the their cloud very active since there are no subfolders it becomes very fast very crowded.

**Reply by bios4** [reply]
Additional note: if you export & store your routes as .kurviger files, the routes will be loaded exactly the same when you open them again (i.e. without any re-routing because of a different algorithm).

This is a major difference to .gpx

**Reply by 0709wiwiwi** [reply]
Additional note: 
Cruiser: If you export & store your routes as:
**KURVIGER**
**JSON**
**GPX navigation**  
_The file format additionally (can) support simple (poi)wpt transfers.
Locus @ import navigates 100% faithful by this file mode. 
Any other app do display the file as a simple (clean) track._

Extra to be expected. (Not yet)
**GPZ navigation.** 
_Compressed gpx format, inclusive simple (poi)wpt, inclusive files (doc pdf images sound) plus any icon transfer._

Routes will be loaded exactly the same when you open them again (i.e. without any re-routing because of a different algorithm).

**Reply by DIrgi** [reply]
*.gpx file was just a common placeholder for all kind of files. Since Cruiser can handle kurviger there‘s no need to use a gpx format but….gpx is the mother of all navigation formats. Every device can normally handle this. So if you‘ll share with others or use on other software you might to transfer them first.

**Reply by devemux86** [reply]
@DIrgi 

This is GPX **navigation** format, which can store **offline** a complete route with navigation instructions and data.
It is supported by BRouter, Cruiser, Locus, [RouteYou website](https://www.routeyou.com/), etc.

Please see:
- https://github.com/devemux86/cruiser/discussions/250

**Reply by 0709wiwiwi** [reply]
@DIrgi   _Info to show in the Cruiser wiki  ?_ 

**True 100% faithfull tracknavigation transfers**.

KURVIGER: 
_Cruiser <+> Kurviger web._

JSON: 
_Cruiser_ 

GeoJSON: 
_BRouter web > Cruiser._

GPX Navigation: 
_Cruiser > Locus._
_BRouter web > Cruiser._
_RouteYou web > Cruiser._
